config/config.exs

```diff
@@ -19,6 +19,20 @@ config :logger, :console,
   metadata: [:request_id, :room_id]
 
 config :phoenix, :json_library, Jason
+config :phoenix, :logger, false
+
+config :logger,
+  compile_time_purge_matching: [
+    [module: Membrane.RTC.Engine, level_lower_than: :warning],
+    [module: Membrane.RTC.Engine.Endpoint.RTSP, level_lower_than: :warning],
+    [module: Membrane.RTC.Engine.Endpoint.HLS, level_lower_than: :warning],
+    [module: Membrane.RTC.Engine.Endpoint.WebRTC, level_lower_than: :warning],
+    [module: Membrane.RTC.Engine.Tee, level_lower_than: :warning],
+    [
+      module: Membrane.RTC.Engine.Endpoint.WebRTC.RTPConnectionAllocator,
+      level_lower_than: :warning
+    ]
```

Comment:
* Will we log now information about new incoming tracks? If engine emits such message we could catch it in Jellyfish and log it on our side instead of relying on Engine
* AFAIK there should be an option to remove all logs from specific application so that we don't have to list all modules
* I think that some of the logs in the Engine like those related to simulcast bandwidth estimation should be moved to debug anyway

Reply:
As for the last issue [PR for this is created](https://github.com/jellyfish-dev/membrane_rtc_engine/pull/327).
As for other issues they are applied.